### PR TITLE
Consistently use option vice switch/flag

### PR DESCRIPTION
**Pull request by tbert**
It may be better to stick with a single term
consistently throughout the text, and mention
that alternate terms may be used by other
shell users in real-world settings.

Also use alternate wording to avoid using "options"
in two different contexts to further avoid confusion.

### CODE REVIEW

File: _episodes/02-filedir.md [hunk 1]
```diff
@@ -734,14 +733,18 @@ $ ls -F /
...
-We've already encountered options (also called **switches** or **flags**) which
+We've already encountered options  which
 either start with a single dash (`-`) or two dashes (`--`),
 and they change the behavior of a command.
 [Arguments] tell the command what to operate on (e.g. files and directories).
 Sometimes options and arguments are referred to as **parameters**.
 A command can be called with more than one option and more than one argument, but a
 command doesn't always require an argument or an option.
 
+Note that options are also called **switches** or **flags**, especially for options
```
Comment (gcapes) [inline _episodes/02-filedir.md:744]: This wording implies that the terms switch and flag are on equal footing with option. I think we're trying to present 'option' as the preferred term to use (because that's what the man pages use), so I would reword to reflect this. Something like
> You might sometimes see  *options* being referred to as **switches** or **flags**, especially for options that take no argument. In this lesson we will stick with using the term *option*.

File: reference.md [hunk 2]
```diff
@@ -120,6 +112,14 @@ operating system
 :   Software that manages interactions between users, hardware, and software [processes](#process).
     Common examples are Linux, macOS, and Windows.
 
+option
+:   A way to specify an argument or setting to a command-line program.
+    By convention Unix applications use a dash followed by a single letter,
+    such as `-v`, or two dashes followed by a word, such as `--verbose`,
+    while DOS applications use a slash, such as `/V`.
```
Comment (gcapes) [inline reference.md:119]: I'm not sure we need to go into DOS in the Unix shell lesson.

File: reference.md [hunk 3]
```diff
@@ -82,14 +82,6 @@ filter
     they read data from [standard input](#standard-input),
     process it, and write the result to [standard output](#standard-output).
 
-flag
-:   A terse way to specify an option or setting to a command-line program.
```
Comment (gcapes) [inline reference.md:86]: This is mostly repeated from the option entry, but without a definition of the difference. I would probably drop this entry, and maybe mention in the option entry this alternative terminology.

**Review by gcapes — CHANGES_REQUESTED** [review]
Thanks for the PR! I'm always keen on contributions which add clarity :)
This PR is good stuff, but I've a few small comments that need addressing please.

File: reference.md (hunk 3 above)
Comment (tbert) [reply reference.md:86]: This entry was renamed to 'option' and slightly cleaned of subjective ("terse way") wording.

File: reference.md (hunk 2 above)
Comment (tbert) [reply reference.md:119]: I more or less just moved the existing text; that change is, IMO, orthogonal to this diff.

File: _episodes/02-filedir.md (hunk 1 above)
Comment (tbert) [reply _episodes/02-filedir.md:744]: I've no objection to this change being made before merge, but doing so via the pull request resolution interface on github is, at present, beyond my ken.

File: reference.md (hunk 3 above)
Comment (gcapes) [reply reference.md:86]: Sorry, I was reading this with tired eyes and got confused, thinking there were not yet entries.

File: reference.md (hunk 2 above)
Comment (gcapes) [reply reference.md:119]: Yes, fair point!

File: _episodes/02-filedir.md (hunk 1 above)
Comment (gcapes) [reply _episodes/02-filedir.md:744]: Let me know if you'd like me to make that change, but the normal approach would be for you to make another commit on your feature branch and push it. That will then your the PR.

Comment (tbert) [reply _episodes/02-filedir.md]: Follow-up commit pushed to branch in forked repo. Was afraid my reliance on command-line tools would force me to learn yet another web interface for a single action.

File: reference.md (hunk 2 above)
Comment (tbert) [reply reference.md:119]: As a follow-up, this may be a good opportunity to flag something for "good first commit" status for purposes of onboarding.

File: _episodes/02-filedir.md (hunk 1 above)
Comment (gcapes) [reply _episodes/02-filedir.md]: I've squashed and merge this PR now. Thanks again for the contribution!
